common/locales/en/tasks.json

```diff
@@ -17,6 +17,7 @@
     "checklistText": "For Dailies, partially completed checklists reduce damage. For example, a 4-item checklist with 3 completed reduces damage for that incomplete Daily to 25%. Completed checklist items on a To-Do add a multiplier: 3 checks means +3x (total 4x) the Experience, Gold, and Mana.",
     "expandCollapse": "Expand/Collapse",
     "text": "Text",
+    "title": "Title",
```

Comment:
It might not be necessary to add the new `title` item here. If `text` is used only for the titles of the text field, then you can just change the word "Text" to "Title". One of the benefits of doing it that way is that all existing translations for `text` will continue to be used (they will be marked in Transifex as needing retranslation, but until that's done, the old translations will keep on being seen by the players, which is better than them seeing the English word "Title"). Can you check to see if `text` is used for anything else? If you're not sure how, say so and I'll check.